{
    "title": "IDAPython: error executing init.py:",
    "link": "https://reverseengineering.stackexchange.com/questions/29193/idapython-error-executing-init-py",
    "content": "<html><body><div class=\"s-prose js-post-body\" itemprop=\"text\">\n <p>\n  When I open Ida Pro 7.3 I see an error message\n </p>\n <pre><code>IDAPython: error executing init.py:\nnothing to repeat\n\nRefer to the message window to see the full error log.\n</code></pre>\n <p>\n  How can I understand what is the problem? in message windows I not see any problem.\n </p>\n <p>\n  I have python in PATH .and only 1 installation of python\n </p>\n</div>\n</body></html>",
    "votes": "0",
    "answers": 1,
    "views": "4k",
    "tags": [
        "ida",
        "idapython"
    ],
    "user": "error",
    "time": "Aug 24, 2021 at 19:59",
    "comments": [
        {
            "user": "user41283",
            "text": "<html><body><span class=\"comment-copy\">\n Have u found the answer yet ?\n</span>\n</body></html>",
            "time": null
        },
        {
            "user": "raspiduino",
            "text": "<html><body><span class=\"comment-copy\">\n Can you provide the\n <code>\n  init.py\n </code>\n ? Does this error come from any extension?\n</span>\n</body></html>",
            "time": null
        },
        {
            "user": "raspiduino",
            "text": "<html><body><span class=\"comment-copy\">\n If it is the\n <code>\n  init.py\n </code>\n in\n <code>\n  [IDA_DIRECTORY]/python/[2 or 3]/init.py\n </code>\n , then maybe your IDA installation is broken\n</span>\n</body></html>",
            "time": null
        }
    ],
    "answers_data": [
        {
            "content": "<html><body><div class=\"s-prose js-post-body\" itemprop=\"text\">\n <p>\n  Read READMY_python3.txt in the folder where IDA is installed.\nFor example, in my case on Windows, running idapyswitch.exe and selecting one of the installed Python distributions helped.\n </p>\n</div>\n</body></html>",
            "votes": "0",
            "user": "Maksim Osminin",
            "time": "Feb 9, 2024 at 8:10",
            "is_accepted": false,
            "comments": []
        }
    ]
}